tests/src/main/java/ma/glasnost/orika/test/community/Issue176WithSuperClassesTestCase.java

```diff
@@ -34,7 +33,7 @@ public B create(Object source, MappingContext mappingContext) {
                 b = new B1();
             } else if ("2".equals(a.type)) {
                 b = new B2();
-            } else if (StringUtils.isEmpty(a.type)) {
+            } else if (a.type == null || a.type.isEmpty()) {
```

Comment:
idk how this worked before without the null check.  
Type never gets set (a0 around line 80), so for this test case it's always null.